spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/bridge/CompositeSpanHandler.java

```diff
@@ -35,8 +36,11 @@
 
 	private final List<SpanFilter> exporters;
```

Comment:
I know this is not really in scope for this PR but it is a little related and seeing `exporters` and `reporters` puzzles me a little? Can we rename `exporters` to `filters`?